.changes/next-release/feature-AmazonDynamoDBEnhancedClient-2d32cd6.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "Amazon DynamoDB Enhanced Client",
+    "contributor": "",
+    "description": "Added support for fluent setters in bean descriptors when the default setter is missing."
```

Comment:
This is a little too internal facing. Can we frame it for the perspective of the user? e.g. "Add support in BeanTableSchema for beans that use fluent setters".